.github/workflows/issue-create-automate-message.yml

```
@@ -0,0 +1,28 @@
+          script: |
+            const issueNumber = context.issue.number;
+            const commentBody = `### Thank you for raising this issue!\n We'll review it as soon as possible. We truly appreciate your contributions! ✨\n\n> Meanwhile make sure you've visited the README.md, CONTRIBUTING.md, and CODE_OF_CONDUCT.md before creating a PR for this. Also, please do NOT create a PR until this issue has been assigned to you. 😊`;
+
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issueNumber,
+              body: commentBody
+            });
+
+            console.log('Comment added successfully.');
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)